lib/stellar_core_commander/docker_process.rb

```diff
@@ -154,6 +155,22 @@ def dump_logs
       @stellar_core_container.logs
     end
 
+    Contract None => Any
+    def dump_system_stats
+      fname = "#{working_dir}/../proc-stats-#{Time.now.to_i}-#{rand 100000}.sql"
+      uptime = IO.read("/proc/uptime").split[0]
+      idletime = IO.read("/proc/uptime").split[1]
```

Comment:
does this blow up on other platforms than Linux? (if so, maybe we should skip on exception)

Reply:
It does, that's why I added `rescue nil` in the end. I'm now thinking that I should add an extra logging statement to clarify why it was skipped.